locallib.php

```diff
@@ -229,7 +229,7 @@ public function webservice_call($function, $params, $method="POST") {
         );
 
         $client = new mahara_oauth($args);
-        if ($CFG->disablesslchecks) {
+        if (isset($CFG->disablesslchecks) && $CFG->disablesslchecks) {
```

Comment:
This can be written as if (!empty($CFG->disablesslchecks))

Reply:
Thank you so much Dima for giving this !!!

Updated and please review.

Thanks,
Anu